### DISCUSSION TITLE
need help saving images

### DISCUSSION THREAD

**Discussion by Tasaddar12** [alandefreitas/matplotplusplus#97] [Q&A]
I want to create a scatter plot of data to save as an overlay for another background image. I dont understand anyway I can save an overlay of data points that will also be the same ratio as my other image. Please any idea how to work around this?

**Comment by alandefreitas** [comment]
https://alandefreitas.github.io/matplotplusplus/examples/appearance/multiplot/hold.html
and then
https://alandefreitas.github.io/matplotplusplus/examples/exporting.html
